scripts/lib/CIME/case/case_run.py

```diff
@@ -250,6 +251,33 @@ def case_run(self, skip_pnl=False):
     # set up the LID
     lid = new_lid()
 
+    resubmit_at_start = False
+
+    if self.get_value("RESUBMIT_AT_START"):
```

Comment:
This should use the RESUBMIT arg to queue all the jobs that will need to be run - that is there should be a loop here from 1 to RESUBMIT and the RESUBMIT variable should be decremented for each job submitted.

Reply:
Yes this should be in case_submit.py.   I'm not sure I understand your comment about CONTINUE_RUN, this change shouldn't have any affect on that variable.

Reply:
I guess my mental model of what happens isn't right?
I thought CONTINUE_RUN was used to set a namelist variable for determining if restarts are to be used; and that namelists are generated in case.run? So setting it in case_submit.py would cause the namelists to be wrong for the first run.
OTOH, the bash script to queue resubmits doesn't have this problem, so I guess I've misunderstood something.